org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java

```diff
@@ -371,4 +377,20 @@ private void cleanUpAWSResources(String connectionId, AmazonApiGatewayManagement
       Logger.getLogger(MAIN_LOGGER).removeHandler(allHandlers[i]);
     }
   }
+
+  private void verifyApiClient(String connectionId) {
+    GetConnectionRequest connectionRequest =
```

Comment:
Should we log a warning/info here just to see if it's still happening, since we don't exactly know the root cause?

Reply:
good call, I'll add a warning log